sources/updater/updog/src/main.rs

```diff
@@ -210,10 +214,17 @@ fn retrieve_migrations(
         fs::create_dir(&dir).context(error::DirCreate { path: &dir })?;
     }
 
+    // find the list of migrations in the manifest based on our from and to versions.
+    let mut targets = find_migrations(start, target, &manifest)?;
+
+    // DEPRECATED CODE BEGIN ///////////////////////////////////////////////////////////////////////
+    // write unsigned migrations for backward compatibility. note that signed migrations will have
```

Comment:
It might also be worth a note in here about how the compat migrations are deflated before writing, where the signed migrations are not, so they can be validated as they were in the repo?

Reply:
Addressed in [4f34544](https://github.com/bottlerocket-os/bottlerocket/compare/afc4b5d9ddfb7c6fb24f30abdaf6a5a000df222c..4f345446016e314c3da3f0855c92db5ca979f2a4)

Reply:
Addressed in [4f34544](https://github.com/bottlerocket-os/bottlerocket/compare/afc4b5d9ddfb7c6fb24f30abdaf6a5a000df222c..4f345446016e314c3da3f0855c92db5ca979f2a4)